docs/content/_navigation.json

```diff
@@ -609,6 +609,10 @@
                     "title": "Microsoft Teams",
                     "path": "/dagster-cloud/managing-deployments/alerts/microsoft-teams"
                   },
+                  {
+                    "title": "PagerDuty",
```

Comment:
We shouldn't list this page until the integration is available, so we can either
- remove it from the nav so it's unlisted, but we can land this diff
- wait to land once the integration is ready, and use the vercel link for the pagerduty application

up to you

Reply:
I'll take it out of the nav + category pages for now